### DISCUSSION TITLE
support i18n

### DISCUSSION THREAD

**Discussion by lanch** [langfuse/langfuse#4855] [Support] [answered]
the project supports english only?

**Comment by marcklingen** [comment, marked as answer]
Yes, Langfuse is currently available only in English to optimize development speed. You can use browser-based translation to access Langfuse in other languages. Does that work for you?

**Reply by svensxl** [reply]
Is there a plan to support i18n recently?  we are looking forward to this ability, if no maybe we can contribute it

**Reply by marcklingen** [reply]
This is currently not a priority as this would decrease development speed. Does in-browser translation work for you? Which kinds of i18n problems do you try to solve?

**Reply by svensxl** [reply]
> This is currently not a priority as this would decrease development speed. Does in-browser translation work for you? Which kinds of i18n problems do you try to solve?

I need to deploy locally so that more people can easily understand and use it, and they cannot use browser translation

**Reply by chenyong** [reply]
When we talk about 'optimizing development speed', is the concern that simultaneously adding new features and supporting multiple languages will slow us down? Or is it that i18n solutions themselves consume significant time for coding and refactoring existing code? My personal view is that Langfuse should simply provide the solution, and users should handle the multi-language setup themselves during deployment.

**Reply by YunKuiLu** [reply]
Browser-based translation is causing errors on the page. I created an issue to track this https://github.com/langfuse/langfuse/issues/8703
<img width="2932" height="1494" alt="image" src="https://github.com/user-attachments/assets/c0d2d2a6-5539-4e51-80bb-8bac71abda16" />

**Comment by AkioNuernberger** [comment]
I've got a similar request for JP.

**Comment by magi-wang** [comment]
i18n please ～
